e2e/bare-client-auth/README.md

```diff
@@ -0,0 +1,3 @@
+# Bare Flower testing
+
+This directory is used for testing Flower in a bare minimum scenario, that is, with a dummy model and dummy operations. This is mainly to test the core functionnality of Flower independently from any framework. It can easily be extendended to test more complex communication set-ups.
```

Comment:
```suggestion
This directory is used for testing Flower in a bare minimum scenario, that is, with a dummy model and dummy operations. This is mainly to test the core functionality of Flower independently from any framework. It can easily be extended to test more complex communication set-ups.
```